packages/components/src/components/as-histogram-widget/interfaces.d.ts

```diff
@@ -1,6 +1,6 @@
 export interface HistogramData {
-  start: number;
-  end: number;
+  start: Date | number;
```

Comment:
This is nitpicky, and I wouldn't really try changing it, but this is tecnically incorrect. If start is a Date, end must be a date too.

The solution is probably making HistogramData a generic HistogramData<T>, but again, don't even bother imho.